dockstore-common/src/main/java/io/dockstore/common/RegressionTest.java

```diff
@@ -0,0 +1,22 @@
+/*
+ *    Copyright 2017 OICR
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package io.dockstore.common;
+
+/**
+ * Setting aside tests that require our confidential data bundles
```

Comment:
Not sure what's going on here. 
We already have ConfidentialTest for tests that require confidential data bundles